librarian-puppet.gemspec

```
@@ -27,7 +27,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency "librarianp", ">=0.6.3"
   s.add_dependency "rsync"
-  s.add_dependency "puppet_forge", ">= 2.1", '< 4'
+  s.add_dependency "puppet_forge", ">= 4.0.0"
```

I'd still prefer to have some semver pinning:
```suggestion
  s.add_dependency "puppet_forge", "~> 4.0"
```

Though I do wonder about keeping support for the older versions.